src/services/api/rentService.ts

```diff
@@ -1,15 +1,49 @@
 import axios from '../../axios/axiosConfig';
 
+const getAllNotReturnedRentedBooks = () => {
+  return axios
+    .get('/rent/book', {
+      params: {
+        current: 'true',
+        page: 0,
+        size: 10,
+        sort: 'id'
+      }
+    })
+    .then((res) => res.data);
+};
+
+const cancelRent = (id: string) => {
+  return axios.put(`/rent/close/${id}`).then((res) => res);
+};
+
+const extendRent = (id: string, days: string) => {
+  return axios.put(`/rent/${id}?additionalRentPeriod=${days}`).then((res) => res);
+};
+
 const rentBook = (id: string, days: string) => {
-  console.log('ID ' + id);
-  console.log('DAYS ' + days);
+  return axios.post(`/rent/book/${id}/user?rentPeriod=${days}`).then((res) => res);
+};
 
+const getUserRentedBooks = (id: string, current: string) => {
   return axios
-    .post(`/rent/book/${id}/user?rentPeriod=${days}`)
-    .then((res) => res)
-    .catch((err) => err);
+    .get(`/rent/user/${id}`, {
+      params: {
+        current: current,
+        page: 0,
+        size: 10,
+        sort: 'id'
+      }
+    })
+    .then((res) => res.data);
 };
 
-const rentService = { rentBook };
+const rentService = {
+  getAllNotReturnedRentedBooks,
+  cancelRent,
+  extendRent,
+  rentBook,
+  getUserRentedBooks
+};
 
 export default rentService;
```

Comment:
Please export each service function separately and remove this const and default export